velox/exec/PartitionedOutputBufferManager.h

```diff
@@ -193,15 +287,28 @@ class PartitionedOutputBufferManager {
       int numDestinations,
       int numDrivers);
 
+  void initializeTask(
```

Comment:
Why do we need 2 initializeTask methods?

Reply:
Ah, we don't need it. I thought it is for Presto cpp. Thanks pointing out!